tests/opentelemetry-docker-tests/tests/prometheus-remote-write-cortex/prometheus_remote_write_cortex_test.py

```diff
@@ -0,0 +1,102 @@
+from opentelemetry import metrics
```

Comment:
file should have the copyright at the top

Reply:
oops, will add!